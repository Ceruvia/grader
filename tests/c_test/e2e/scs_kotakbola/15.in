884 924 697
B 74
M 82
B 33
M 6
H 60
M 34
H 54
M 2
M 98
H 61
B 49
M 69
M 16
B 1
M 66
M 12
H 10
B 90
M 96
H 16
H 39
H 32
H 54
M 37
B 52
H 25
M 15
H 33
H 21
H 65
M 20
M 67
M 69
M 84
B 24
B 43
B 15
B 83
H 68
M 73
E
