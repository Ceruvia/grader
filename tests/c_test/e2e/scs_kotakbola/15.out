538 846 464
Ya
